stripe/src/main/java/com/stripe/android/PaymentIntentResult.kt

```diff
@@ -10,5 +10,6 @@ import kotlinx.android.parcel.Parcelize
 @Parcelize
 data class PaymentIntentResult internal constructor(
     override val intent: PaymentIntent,
-    @Outcome private val outcomeFromFlow: Int = 0
+    @Outcome private val outcomeFromFlow: Int = 0,
+    val context: String? = null
```

Comment:
`failureMessage`?

Reply:
Naming is always hard. I had originally called this something like `failureMessage` but thought that may be too limiting is scope. I guess it's best to be specific about what this will be used for though